app/services/platforms/mixer.ts

```diff
@@ -186,7 +186,7 @@ export class MixerService extends BasePlatformService<IMixerServiceState>
     ).then(json => json.viewersCurrent);
   }
 
-  async putChannelInfo({ title, game }: IMixerStartStreamOptions): Promise<boolean> {
+  async putChannelInfo({ title, game }: IMixerStartStreamOptions): Promise<void> {
```

Comment:
I think technically speaking we could just get rid of this whole service right?

Reply:
Yes, we can. I prefer to not do it as a part of this PR